app/main/views/marketplace.py

```diff
@@ -269,6 +269,13 @@ def get_brief_by_id(framework_slug, brief_id):
     application_url = "/sellers/opportunities/{}/responses/create".format(brief['id'])
     add_case_study_url = None
 
+    # pass application status to brief
+    profile_application_status = None
+    if current_user.is_authenticated and current_user.role == 'supplier':
+        profile_application_id = current_user.application_id
```

Comment:
for a supplier I would not rely on user.application_id, I would do back to application table and query on supplier_code